templates/index.html

```diff
@@ -473,6 +473,7 @@ <h2 class="p-modal__title">Get in touch</h2>
       <input type="hidden" name="utmcontent" id="utmcontent" value="">
       <input type="hidden" name="utm_term" id="utm_term" value="">
       <input type="hidden" name="GCLID__c" id="GCLID__c" value="">
+      <input type="hidden" aria-hidden="true" aria-label="hidden field" name="acquisition_url" id="acquisition_url" value="{{ request.url }}" />
```

Comment:
nit: hidden fields don't need ARIA labels

```suggestion
      <input type="hidden" name="acquisition_url" id="acquisition_url" value="{{ request.url }}" />
```